opentargets.json

```diff
@@ -1181,9 +1254,19 @@
         "animal_model",
         "somatic_mutation",
         "known_drug",
-        "genetic_association"
+        "genetic_association",
+        "ot_partner"
       ]
     },
+    "direction": {
```

Comment:
If this is something specific to CRISPR studies, could we make the field name more specific, for example `crisprDirection`? Or are we expecting to reuse it in other datasources?